tests/testcases/Group9-Content-trust/9-01-DB-user-push-signed-images.md

```diff
@@ -3,21 +3,26 @@ Test 9-01 User push signed images(DB mode)
 
 # Purpose:
 
-TBD
+To verify user can push images with content trust enabled.
 
 # References:
 User guide
 
 # Environment:
 
+* This test requires one Harbor instance is runnning and available.
+* A Linux host with Docker CLI installed (Docker client).
 
 # Test Steps:
 
-TBD
+1. Login UI and create a project.
+2. On Docker clinet, set DOCKER_CONTENT_TRUST and DOCKER_CONTENT_TRUST_SERVER properly and login Harbor.
```

Comment:
I think you should elaborate how to set the env properly, at least by referencing other docs.